templates/leaflet/index.html.twig

```diff
@@ -0,0 +1,20 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Hello LeafletController!{% endblock %}
+
+{% block body %}
+<style>
+    .example-wrapper { margin: 1em auto; max-width: 800px; width: 95%; font: 18px/1.5 sans-serif; }
+    .example-wrapper code { background: #F5F5F5; padding: 2px 6px; }
+</style>
+
+<div class="example-wrapper">
+    <h1>Hello {{ controller_name }}! ✅</h1>
+
+    This friendly message is coming from:
+    <ul>
+        <li>Your controller at <code><a href="{{ '/home/zurabi/Documents/WildCS/Projet3/orleans-202103-php-project-jobpermut/src/Controller/LeafletController.php'|file_link(0) }}">src/Controller/LeafletController.php</a></code></li>
+        <li>Your template at <code><a href="{{ '/home/zurabi/Documents/WildCS/Projet3/orleans-202103-php-project-jobpermut/templates/leaflet/index.html.twig'|file_link(0) }}">templates/leaflet/index.html.twig</a></code></li>
+    </ul>
+</div>
+{% endblock %}
```

Comment:
this template is useless

Reply:
yes you are right, i didn't looked there was tamplate.